optionals/compat_rhs_afrf3/CfgAmmo.hpp

```diff
@@ -126,4 +126,66 @@ class CfgAmmo
     class rhs_ammo_127x108mm_x5: SubmunitionBase {
         ACE_rearm_caliber=13;
     };
+    
+    class GrenadeHand;
+    class rhs_ammo_rgd5: GrenadeHand {
+	    ace_frag_enabled = 1;
+        ace_frag_metal = 200;
+        ace_frag_charge = 110;
+        ace_frag_gurney_c = 2440;
+        ace_frag_gurney_k = "3/5";
+        ace_frag_classes[] = {"ACE_frag_small_HD"};
+        ace_frag_skip = 0;
+        ace_frag_force = 1;
+    };
+    class rhs_ammo_rgn_base: rhs_ammo_rgd5 {
+	    ace_frag_enabled = 1;
+        ace_frag_metal = 193;
+        ace_frag_charge = 97;
+        ace_frag_gurney_c = 2800;
+        ace_frag_gurney_k = "3/5";
+        ace_frag_classes[] = {"ACE_frag_tiny_HD"};
+        ace_frag_skip = 0;
+        ace_frag_force = 1;
+    };
+    class rhs_ammo_rgn: rhs_ammo_rgn_base {
+        ace_frag_enabled = 0;
+        ace_frag_skip = 1;
+        ace_frag_force = 0;
+    };
+    class rhs_ammo_rgn_sub: rhs_ammo_rgn_base {};
+    class rhs_ammo_rgn_exp: rhs_ammo_rgn_base {};
+    class rhs_ammo_fakel: GrenadeHand {
+        ace_frag_enabled = 0;
+        ace_frag_skip = 1;
+        ace_frag_force = 0;
+    };
+    class rhs_ammo_fakels: rhs_ammo_fakel {};
+    class rhs_ammo_zarya2: rhs_ammo_fakels {};
+    class rhs_ammo_plamyam: rhs_ammo_fakels {};
+	
+    class RocketBase;
+    class R_PG32V_F: RocketBase {};
+    class rhs_rpg26_rocket: R_PG32V_F {};
+    class rhs_rpg7v2_pg7vl: rhs_rpg26_rocket {};
+    class rhs_rpg7v2_og7v: rhs_rpg7v2_pg7vl {
+        ace_frag_enabled = 1;
+        ace_frag_metal = 400;
+        ace_frag_charge = 210;
+        ace_frag_gurney_c = 2800;
+        ace_frag_gurney_k = "3/5";
+        ace_frag_classes[] = {"ACE_frag_medium_HD"};
+        ace_frag_skip = 0;
+        ace_frag_force = 1;
+    };
+    class rhs_rpg7v2_tbg7v: rhs_rpg7v2_pg7vl {
+        ace_frag_enabled = 0;
```

Comment:
I got permission from this mod author to add this in. 
https://forums.bistudio.com/topic/185587-ace3xrhs-fragmentation-configs/ 
its a carbon copy of that with Spaces instead of Tabs.